edscrapers/tools/dashboard/pages/insights.py

```diff
@@ -148,8 +149,9 @@ def dataset_by_domain_bar(self):
         """ function creates a bar chart which displays the
         number of pages/datasets per domain """
         # the the dataframe from the Excel sheet
-        #df = self._get_df_from_excel_sheet('PAGE COUNT (DATOPIAN)')
+        #df = self._get_df_from_excel_sheet('PAGE COUNT')
```

Comment:
This is the refactored code but I had to comment it to comply with the changes Higor made